docker/src/main/java/org/arquillian/cube/impl/client/CubeConfiguration.java

```diff
@@ -133,8 +133,8 @@ public static CubeConfiguration fromMap(Map<String, String> map) {
             cubeConfiguration.autoStartContainers = ConfigUtil.trim(map.get(AUTO_START_CONTAINERS).split(","));
         }
 
-        if(map.containsKey(SHOULD_ALLOW_TO_CONNECT_TO_RUNNING_CONTAINERS)) {
-            cubeConfiguration.shouldAllowToConnectToRunningContainers = Boolean.parseBoolean(map.get(SHOULD_ALLOW_TO_CONNECT_TO_RUNNING_CONTAINERS));
+        if(map.containsKey(CONNECTION_MODE)) {
+            cubeConfiguration.connectionMode = ConnectionMode.valueOf(ConnectionMode.class, map.get(CONNECTION_MODE));
```

Comment:
Add some better Exception handling then just the default IllegalArgumentException("No Enum constant " + type...) coming from valueOf